DP3TApp/Screens/Onboarding/NSOnboardingDisclaimerViewController.swift

```diff
@@ -68,7 +76,57 @@ class NSOnboardingDisclaimerViewController: NSOnboardingContentViewController {
 
         privacyButton.contentHorizontalAlignment = .center
 
+
+
+        func addSpacer(spacing: CGFloat? = nil) {
+            let spacer = UIView()
+            spacer.backgroundColor = .ns_text_secondary
+            addArrangedView(spacer, spacing: spacing)
+            spacer.snp.makeConstraints { make in
+                make.width.equalTo(self.stackScrollView.stackView)
+                make.height.equalTo(1)
+            }
+        }
+
+        addSpacer()
+
+        addArrangedView(privacyHeader)
+        privacyHeader.snp.makeConstraints { make in
+            make.width.equalTo(self.stackScrollView.stackView)
+        }
+        addArrangedView(privacyBody, spacing: NSPadding.large)
+        privacyBody.snp.makeConstraints { make in
+            make.width.equalTo(self.stackScrollView.stackView)
+        }
+        privacyBody.superview?.isHidden = true
+        privacyHeader.didExpand = { [weak self] expanded in
+            guard let self = self else { return }
+            self.privacyBody.superview?.isHidden = !expanded
+            self.privacyButton.superview?.isHidden = !(self.privacyHeader.isExpanded || self.conditionOfUseHeader.isExpanded)
+        }
+
+        addSpacer()
+
+        addArrangedView(conditionOfUseHeader)
+        conditionOfUseHeader.snp.makeConstraints { make in
+            make.width.equalTo(self.stackScrollView.stackView)
+        }
+        addArrangedView(conditionOfUseBody, spacing: NSPadding.large)
+        conditionOfUseBody.snp.makeConstraints { make in
+            make.width.equalTo(self.stackScrollView.stackView)
+        }
+        conditionOfUseBody.superview?.isHidden = true
```

Comment:
can you leave a comment to why superviews are used instead of directly the view.